src/lines.js

```diff
@@ -530,7 +530,7 @@ export default function lines(id) {
       let _data = data.map(item => Object.assign(
             {},
             item,
-            {v: item.v.map((v, idx) => legendsEnabled.includes(idx) ? v : null)}
+            {v: item.v.map((v, idx) => !legend.length || legendsEnabled.includes(idx) ? v : null)}
```

Comment:
As on the PR description, if there are no legends (only data is provided), still count `v` as a valid value.